config.go

```diff
@@ -8,7 +8,8 @@ import (
 
 var DefaultConfig = Config{
 	Jobs: jobsConfig{
-		RetryInterval: 60,
+		RetryInterval: "PT1M",
```

Comment:
> Is there a specific reason to change RetryInterval to ISO duration?

It had something to do with how the data was being passed and compared between functions and so on.  Ah yes, I was changing `Table.UpdateNextRun()` to take an ISO8601 duration, but I found that the same code path was used for a successful and a failed execution, so RetryInterval would need to feed into the same argument.

It just seemed silly to have `Table.UpdateNextRun()` have *both* an ISO8601 and regular `time.Duration` and use one or the other.  I suppose I could do a conversion from `time.Duration` into ISO8601 before passing it in?

> A user can start to use Dalga without knowing about ISO durations?

Separate from the above, I'm not sure I understand.  Won't he need to know ISO8601 durations now because that's how you schedule regular intervals?  Unless you're thinking of one-off jobs specifically.

> While we are doing a major version bump, I would also like to change the config file format to TOML, it is more common nowadays. github.com/BurntSushi/toml library has also ability to unmarshal time.Duration values. Do you have any comments on this?

Heh, personally I don't like TOML.  Every time I go to read up on the documentation, I get to the part of how you can use `[[something]]` or `something.something` and they're basically the same except they're not, and I get confused and abandon the attempt.

I am a YAML fan, if you're into that.  I find it's intuitive and easy to use that config format, as long as I'm not the one writing the parser for it 😉 

I have used [spf13/viper](https://github.com/spf13/viper) pretty extensively in the past and like it, although it has a lot of dependencies.  I would *probably* only go that way if you considered switching to [spf13/cobra](https://github.com/spf13/cobra) for the CLI framework, since they integrate very well.  But then, I have seen very minimal dependencies and mostly stdlib in Dalga so I doubt you want to go that route.

Something like [knadh/koanf](https://github.com/knadh/koanf) might do, or even just a straight YAML parser.  But that depends on how you feel about YAML.  I am also a fan of HJSON 😄